templates/docs/patterns/badge/accessibility.md

```diff
@@ -23,6 +23,6 @@ This component strives to follow [WCAG 2.1 (level AA) guidelines](https://www.w3
 
 Applicable WCAG guidelines:
 
-- [WCAG21 - Use of color](https://www.w3.org/TR/WCAG21/#use-of-color)
-- [WCAG21 - Sensory characteristics](https://www.w3.org/TR/WCAG21/#sensory-characteristics)
-- [WCAG21 - Abbreviations](https://www.w3.org/TR/WCAG20-TECHS/H28.html)
+- [1.4.1: Use of Color](https://www.w3.org/TR/WCAG22/#use-of-color)
+- [1.3.3: Sensory Characteristics](https://www.w3.org/TR/WCAG22/#sensory-characteristics)
+- [3.1.4: Abbreviations](https://www.w3.org/TR/WCAG22/#abbreviations)
```

Comment:
Since these three are numbered differently now, it would be nice to order them by WCAG section number and/or section title